brave-kafka/src/main/java/smartthings/brave/kafka/consumers/DefaultTracingConsumerInterceptor.java

```diff
@@ -44,7 +57,7 @@
       if (envelope.hasParentId()) {
         builder.parentId(envelope.getParentId().getValue());
       }
-      TraceContextOrSamplingFlags traceContextOrSamplingFlags = TraceContextOrSamplingFlags.create(builder);
+      TraceContextOrSamplingFlags traceContextOrSamplingFlags = TraceContextOrSamplingFlags.create(builder.build());
```

Comment:
make a single-message version of this eventually, if consumer interceptor are still needed.

 https://github.com/openzipkin/brave/blob/master/instrumentation/kafka-clients/src/main/java/brave/kafka/clients/TracingConsumer.java#L60